runtime/ops/worker_host.rs

```diff
@@ -206,33 +207,37 @@ fn op_create_worker(
     // - JS worker is useless - meaning it throws an exception and can't do anything else,
     //  all action done upon it should be noops
     // - newly spawned thread exits
+    let fut = async move {
```

Comment:
This is the fix - the whole creation and running of a worker is done inside an async block - ensuring that we are inside a Tokio runtime